### PR TITLE
Brev init add files

### PR DESCRIPTION
**Pull request by naderkhalil**

### CODE REVIEW

File: internal/brev/utils.go [hunk 1]
```diff
@@ -43,3 +47,77 @@ func GetActiveProject() BrevProject {
 
 	return project
 }
+
+func GetRootDir() string {
+	usr, err := user.Current()
```
Comment (drewmalin) [inline internal/brev/utils.go:52]: Tab vs. space issue?

File: internal/brev/utils.go [hunk 2]
```diff
@@ -43,3 +47,77 @@ func GetActiveProject() BrevProject {
...
+	if (IsInProjectDirectory()) {
+		return true
+	}
+
+	rootDir := GetRootDir()
+	var curr_brev_directories []string
+	files.ReadJSON(rootDir+"/.brev/active_projects.json", &curr_brev_directories)
+
+	// Exit with error message
+	fmt.Println("Endpoint commands only work in a Brev project.")
```
[the remark this answers is not in the document]
Comment (naderkhalil) [reply internal/brev/utils.go:86]: Adding this as a todo for when we handle the error messages

File: internal/brev/utils.go [hunk 3]
```diff
@@ -43,3 +47,77 @@ func GetActiveProject() BrevProject {
...
+	if (len(curr_brev_directories)==0) {
+		// If no directories, check if they have some remote.
+
+		// Get Projects
+		token, _ := auth.GetToken()
```
[the remark this answers is not in the document]
Comment (naderkhalil) [reply internal/brev/utils.go:91]: I like this, I'd even wonder if the API agent should handle that so helper functions can just `BrevAPI.GetEndpoints()`

File: internal/brev/utils.go [hunk 4]
```diff
@@ -43,3 +47,77 @@ func GetActiveProject() BrevProject {
...
+
+		} else {
+			// Encourage them to pull one of their existing projects
+			fmt.Println("Set up one of your existing projects.")
+			fmt.Println("For example, run 'brev init " + raw_projects[0].Name + "'")
+		}
+		
+	} else {
+		// Print active brev projects
+		fmt.Println("Active Brev projects on your computer: ")
```
[the remark this answers is not in the document]
Comment (naderkhalil) [reply internal/brev/utils.go:108]: What does the out writer do? I was gonna look into the proper way to make all these output messages when I do colors/formatting/spinners, etc. If this is the way to do it im down to just start that now.

File: internal/brev/utils.go [hunk 5]
```diff
@@ -43,3 +47,77 @@ func GetActiveProject() BrevProject {
...
+}
+
+func StringInList(a string, list []string) bool {
+    for _, b := range list {
+        if b == a {
+            return true
+        }
+    }
+    return false
+}
```
[the remark this answers is not in the document]
Comment (naderkhalil) [reply internal/brev/utils.go:123]: I think that'd be great. I threw this in bcs I really missed pythons `in` operator haha

File: internal/brev/utils.go (hunk 4 above)
[the remark this answers is not in the document]
Comment (drewmalin) [reply internal/brev/utils.go:108]: There are three writers:

Out
Verbose
Err

By using fmt.Fprintln you can choose exactly which one to use, but the rule of thumb should be:
1. If you know it's an error, write to context.Err (that will use the appropriate STDERR flag)
2. If it's just simple output, write to context.Out (the users's use of the --verbose flag will control whether or not this gets written to STDOUT or is just ignored)
3. If it *must* be seen (not controlled by flags) write to context.VerboseOut

File: internal/brev/utils.go (hunk 1 above)
Comment (naderkhalil) [reply internal/brev/utils.go:52]: My editor is set to tab size 4. Should i make it spaces 4?

Comment (drewmalin) [reply internal/brev/utils.go:52]: Hmm, here it looks like line 52 has a tab, but lines 53 - 56 use 4 spaces (the rest of the PR uses tabs)

**Review by drewmalin — APPROVED** [review]

**Pull request merged**